linuxHelpers.cpp

```diff
@@ -623,7 +623,7 @@ LARGE_INTEGER GetLogonTime(
     CONST LUID* user_luid
     )
 {
-    LARGE_INTEGER result = {0,};
+    LARGE_INTEGER result = {{0},};
```

Comment:
Wouldn't
LARGE_INTEGER result = {0};
make more sense? i.e. initialize the QuadPart? The comma is probably the error here.
But appreciate your input; let me know and I'll approve/merge.

Reply:
That's a good point, I was just trying to mimic the existing initialization behavior while also following what the -Werror hint was telling me. I'll update with the modification.